src/System.CommandLine.Tests/SuggestionTests.cs

```diff
@@ -144,7 +143,7 @@ public void When_one_option_has_been_specified_then_it_and_its_siblings_will_sti
         }
 
         [Fact]
-        public void When_a_subcommand_has_been_specified_then_its_siblings_will_not_be_suggested()
+        public void When_a_subcommand_has_been_specified_then_its_sibling_options_will_not_be_suggested()
```

Comment:
I am not sure this rename is correct. The options are not siblings to the specified subcommand. They are children of the siblings to the specified subcommand (nieces and nephews). Did you mean to have sibling options on the root command?